components/event-bus/internal/knative/subscription/controller/eventactivation/reconcile_test.go

```diff
@@ -25,16 +26,26 @@ import (
 
 const (
 	subUID   = "test-uid"
-	subName  = "my-sub-1"
+	subName  = "my-sub"
 	eaName   = "my-event-activation"
 	sourceID = "my_source_ID"
 
 	testNamespace = "default"
 )
 
 func init() {
-	_ = eventingv1alpha1.AddToScheme(scheme.Scheme)
-	_ = subApis.AddToScheme(scheme.Scheme)
+	err := eventingv1alpha1.AddToScheme(scheme.Scheme)
```

Comment:
Why this was not needed before?

Reply:
I handled errors which were ignored before

Reply:
Actually I mean adding this:
`eventingv1alpha1.AddToScheme(`
`subApis.AddToScheme(`
`messagingV1Alpha1.AddToScheme(`

Reply:
I got you now, `messagingV1Alpha1.AddToScheme` is not needed hence removed.
But rest are needed as following is the relation between scheme vs apis:
`eventingv1alpha1 `  -> EventActivation
`subApis` -> Kyma Subscription